go-controller/pkg/ovn/egressqos_test.go

```diff
@@ -972,3 +1023,20 @@ func createNodeAndLS(fakeOVN *FakeOVN, name, zone string) (*v1.Node, *nbdb.Logic
 
 	return kapiNode, logicalSwitch, nil
 }
+
+func expectEgressQoSStatusMessageEventually(fakeOVN *FakeOVN, namespace string, expectFailure, expectEmpty bool) {
```

Comment:
what does `expectEmpty` mean? when is it used?
You probably also can just check status.status here

Reply:
correct :) The `expectEmpty` is never meant to be used for the tests here. removing it.
These unit tests validates status at zone/node level, so not checking status.status here.